jsk_2016_01_baxter_apc/node_scripts/pipeline_segmentation_in_bin_ros.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+from jsk_apc2016_common.spatial_feat import get_spatial_img
+from jsk_apc2016_common.mask_bin import get_mask_img
+from jsk_apc2016_common.pipeline_segmentation_in_bin\
```

Comment:
https://github.com/start-jsk/jsk_apc/tree/master/jsk_apc2016_common/python/jsk_apc2016_common

pipeline_segmentation_in_bin.py is missing, no?